app/services/notifications/notifiable_action/send.rb

```diff
@@ -42,8 +42,8 @@ def call
       attr_reader :notifiable, :action
 
       def followers
-        followers = notifiable.user.followers
-        followers += notifiable.organization.followers if notifiable.organization_id
+        followers = notifiable.user.followers_scoped.where(subscription_status: "all_posts").map(&:follower)
```

Comment:
`user.followers_scoped` is a method from the gem that eager loads the `followers`, and is a shorthand for `Follow.where(followable: user).includes(:follower)`.